libs/blocks/global-navigation/global-navigation.js

```diff
@@ -157,18 +159,27 @@ const closeOnClickOutside = (e) => {
 };
 
 class Gnav {
-  constructor(body, el) {
+  /**
+   * Gnav constructor
+   * @param {Object} config Configuration object for Gnav class
+   * @param {Element} config.body The raw content to decorate to populate the Gnav
+   * @param {Element} config.el The placeholder element where the Gnav should be rendered
+   * @param {Boolean} [config.useFederatedContent] Whether the Gnav loads from a central location
+   */
+  constructor(config) {
```

Comment:
Yup, I'll do something along those lines. Something I wanted to avoid is renaming the arguments, as they're used throughout the whole file, so expect a large diff of updated names 😅